deps/exokit-bindings/webglcontext/src/webgl.cc

```diff
@@ -4919,6 +4952,8 @@ const char *webglExtensions[] = {
   "OES_texture_half_float",
   "OES_texture_half_float_linear",
   "OES_vertex_array_object",
+  "OVR_multiview2",
```

Comment:
We probably do not want to advertise this if it does not exist -- it might not on desktops.

Reply:
0a608de
and
0cbd3f1

Reply:
The problem is, if we don't advertise it for the case where it does exist, then user code will think it's not supported.

More generally, I think we should not hardcode this extension list, but check each one with the hardware and return the whitelisted WebGL ones.

Reply:
Something like: iterate through the current list and return the whitelisted result here: https://github.com/exokitxr/exokit/blob/add-multiview/deps/exokit-bindings/webglcontext/src/webgl.cc#L4968

Reply:
46be25f